cni/scripts/scale_deployment.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -ex
+kubectl apply -f cni/deploy/pod.yaml
+kubectl apply -f ./cni/deploy/hostprocess.yaml
+sleep 1m
+total_num_of_run=10
+echo "Total num of run $total_num_of_run"
+for ((i=1; i <= total_num_of_run; i++))
+do 
+    echo "Current Run: $i"
+    scale_up_of_pods=2400
+    scale_down_pods=0
+    echo "Scaling pods to : $scale_up_of_pods"
+    kubectl scale deployment container --replicas $scale_up_of_pods
+    sleep 2m
```

Comment:
instead of sleep, check deployment status to confirm all pods are up

Reply:
Updated the PR